Resize fig4

estherrolf opened the pull request:
fig 4 --> 3.5in x 6.1in

bolliger32 commented on api_keys.json:
```diff
@@ -0,0 +1,3 @@
+{
```
Can we remove this file? We don't use the census api anymore (plus don't want to commit any API keys :) )

estherrolf replied:
yeah i thought that was weird -- git wouldn't let me pull from master b/c of a conflict but I can delete

estherrolf commented:
deleted api_keys.json and also changed the way we output data so it's cut off at a reasonable precision -- Hannah was having an issue with the tests failing b/c of rounding issues

bolliger32 commented:
yay! thanks!
